var/spack/repos/builtin/packages/minigmg/package.py

```diff
@@ -24,16 +24,47 @@ class Minigmg(Package):
 
     version('master', sha256='1c2d27496a881f655f5e849d6a7a132625e535739f82575991c511cc2cf899ac')
 
+    version('local', branch='minigmg', git='https://github.com/FindHao/minigmg')
+
+    variant('debug', default=False, description='Enable debug flag')
+    variant('simde', default=False, description='enable simde')
+    variant('opt', default=False,
+            description='enable -Ofast -D__PREFETCH_NEXT_PLANE_FROM_DRAM -D__FUSION_RESIDUAL_RESTRICTION')
```

Comment:
```suggestion
            description='enable optimization flags')
```
This description probably makes more sense to humans.